src/renderer/components/header/Header.util.ts

```diff
@@ -19,20 +16,14 @@ export const headerNetStatusSubheadline = ({
   onlineStatus,
   notConnectedTxt
 }: {
-  url: O.Option<string>
+  url: string
   onlineStatus: OnlineStatus
   notConnectedTxt: string
-}) => {
-  if (onlineStatus === OnlineStatus.OFF) return notConnectedTxt
-  return FP.pipe(
-    url,
-    O.chain(API.getHostnameFromUrl),
-    O.getOrElse(() => notConnectedTxt)
-  )
-}
+}) => (onlineStatus === OnlineStatus.ON ? url : notConnectedTxt)
 
-export type HeaderNetStatusColor = 'green' | 'yellow' | 'red'
-export const headerNetStatusColor = ({ url, onlineStatus }: { url: O.Option<string>; onlineStatus: OnlineStatus }) => {
-  if (onlineStatus === OnlineStatus.OFF) return 'red'
-  return O.isSome(url) ? 'green' : 'yellow'
-}
+export type HeaderNetStatusColor = 'green' | 'red'
```

Comment:
> error -> yellow icon
'yellow'  is still needed, see Details of issue description (part "WANT") https://github.com/thorchain/asgardex-electron/issues/1734

Reply:
Yep, that's fixed just forgot to push it